docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -6,10 +6,11 @@ VIVIDUS configuration includes the following parts: profiles, environments and s
 
 [WARNING]
 ====
-Properties prefixed with `configuration.` could be specified in the following locations only:
+Properties prefixed with `*configuration.*` could be specified in the following locations only:
```

Comment:
mabe "Properties prefixed with `*configuration.*` must be specified only in the following locations"? "could" looks a bot weird here

Reply:
Please consult professionals :)